restapi/dataset/item.avsc

```diff
@@ -4,15 +4,10 @@
   "name": "Item",
   "doc": "Dataset item. It contains a value and its related time.",
   "fields": [
-    {"name": "value", "type": [
-      "org.radarcns.avro.restapi.sensor.HeartRate" ,
-      "org.radarcns.avro.restapi.sensor.Acceleration" ,
-      "org.radarcns.avro.restapi.sensor.Battery" ,
-      "org.radarcns.avro.restapi.sensor.BloodVolumePulse" ,
-      "org.radarcns.avro.restapi.sensor.ElectroDermalActivity" ,
-      "org.radarcns.avro.restapi.sensor.InterBeatInterval" ,
-      "org.radarcns.avro.restapi.sensor.Temperature"
-      ] } ,
-    {"name": "effectiveTimeFrame", "type": "org.radarcns.avro.restapi.header.EffectiveTimeFrame"}
+    {"name": "sample", "type": [
+      "org.radarcns.avro.restapi.data.DoubleSample" ,
```

Comment:
Why use DoubleSample at all, and not just expand to `["double", "Quartile", "Acceleration"]`? I'm not sure that `sample` is a better description than `value`, since the rest API currently does not return samples but only statistics.